deployment/v3/external/postgres/README.md

```diff
@@ -69,8 +69,8 @@ kubectl apply -f postgres-postgresql.yaml
 
 ## DB Upgrade
 * Used to upgrade the schemas and required data from one MOSIP version to another.
-* At a time upgrade server performs upgrade from one sequential version to another.
-* In case of upgrade of multiple versions, do update all the details sequentially in `upgrade.csv`.
+* At a time upgrade server performs upgrade from one DB version to another DB version.
```

Comment:
Upgrade script performs upgrade upgrade from the previous version (n-1) to the current version (n). Follow the upgrade path if you are n-2 away from the current version.

Reply:
Updated the README.md file. @gsasikumar